projects/core/bindaas-core-impl/src/main/java/edu/emory/cci/bindaas/core/jwt/Token.java

```diff
@@ -0,0 +1,44 @@
+package edu.emory.cci.bindaas.core.jwt;
+
+import com.google.gson.annotations.Expose;
+
+public class Token {
```

Comment:
Token object was removed in subsequent commits as the string contains all the necessary information.